idb/setup_postgres.sql

```diff
@@ -29,6 +29,8 @@ PRIMARY KEY ( round, intra )
 
 -- NOT a unique index because we don't guarantee txid is unique outside of its 1000 rounds.
 CREATE INDEX IF NOT EXISTS txn_by_tixid ON txn ( txid );
+-- Optional, to make txn queries by asset fast:
+-- CREATE INDEX CONCURRENTLY IF NOT EXISTS txn_asset ON txn (asset, round, intra);
```

Comment:
Let's get an estimate (size-wise) of this one before we decide to make it optional. It's a basic operation and shipping it with bad performance isn't great.

Reply:
mainnet:
```
9688719360 bytes of txn content
 971038720 bytes of txn_by_txid index 
 507478016 bytes of txn_asset index
 302014464 bytes of (round,offset) primary key
3484016640 bytes of txn_participation join table
```